e2e/stack_test.go

```diff
@@ -600,38 +535,20 @@ func createL1TransactOpts(
 	stack *e2e.StackConfig,
 	user *ecdsa.PrivateKey,
 	l1signer types.Signer,
-	l1GasLimit uint64,
 	value *big.Int,
 ) *bind.TransactOpts {
-	address := crypto.PubkeyToAddress(user.PublicKey)
 	return &bind.TransactOpts{
-		From: address,
+		From: ethcrypto.PubkeyToAddress(user.PublicKey),
 		Signer: func(addr common.Address, tx *types.Transaction) (*types.Transaction, error) {
 			signed, err := types.SignTx(tx, l1signer, user)
 			require.NoError(t, err)
 			return signed, nil
 		},
-		Nonce:    getCurrentUserNonce(t, stack, address),
```

Comment:
Nice that we can remove explicitly setting the nonce! Is this inferred somewhere by default for us?

Reply:
Yes I believe so. I don't remember the exact flow, but it works so I assume the answer is yes.